xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/ExportAction.java

```diff
@@ -197,23 +220,69 @@ private String exportHTML(XWikiContext context) throws XWikiException, IOExcepti
                     for (String docName : docsNames) {
                         String pageReference = wikiName + XWikiDocument.DB_SPACE_SEP + docName;
                         if (context.getWiki().getRightService().hasAccessLevel(
-                            "view", context.getUser(), pageReference, context))
+                                "view", context.getUser(), pageReference, context))
                         {
                             pageList.add(resolver.resolve(pageReference));
                         }
                     }
                 }
             } catch (QueryException e) {
                 throw new XWikiException(XWikiException.MODULE_XWIKI_APP, XWikiException.ERROR_XWIKI_APP_EXPORT,
-                    "Failed to resolve pages to export", e);
+                        "Failed to resolve pages to export", e);
             } finally {
                 context.setWikiId(database);
             }
         }
-
         return pageList;
     }
 
+    /**
+     * Resolve the set of pages to export based on choices made in a paginated tree
```

Comment:
The XWiki code style leaves an empty line here. Did you configure your IDE to use the XWiki code style?